Resolving requested class warning

lharries opened the pull request:
Resolves warning:
```
You have requested document class `Classes/PhDThesisPSnPDF',
           but the document class provides `PhDThesisPSnPDF'.
```

As observered in the Overleaf template

Source for fix: [https://tex.stackexchange.com/questions/354098/you-have-requested-document-class-x-but-the-document-class-provides-y-warning](https://tex.stackexchange.com/questions/354098/you-have-requested-document-class-x-but-the-document-class-provides-y-warning)

johannesbottcher commented:
This is just as wrong as having the documentclass line with a file path
inside. The only right solution is to remove the documentclass from the
folder and putting it next to the main file.

On Fri, 3 Aug 2018, 14:38 Luke Harries, <notifications@github.com> wrote:

> Resolves warning:
>
> You have requested document class `Classes/PhDThesisPSnPDF',
>            but the document class provides `PhDThesisPSnPDF'.
>
> As observered in the Overleaf template
>
> Source for fix:
> https://tex.stackexchange.com/questions/354098/you-have-requested-document-class-x-but-the-document-class-provides-y-warning
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/kks32/phd-thesis-template/pull/170
> Commit Summary
>
>    - Resolving requested class warning
>
> File Changes
>
>    - *M* Classes/PhDThesisPSnPDF.cls
>    <https://github.com/kks32/phd-thesis-template/pull/170/files#diff-0>
>    (2)
>
> Patch Links:
>
>    - https://github.com/kks32/phd-thesis-template/pull/170.patch
>    - https://github.com/kks32/phd-thesis-template/pull/170.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kks32/phd-thesis-template/pull/170>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/AH4EjpynPwZtrT8lL7SAsPZk24nTxogYks5uNERVgaJpZM4Vt6Op>
> .
>

lharries commented:
Is there any reason why the documentclass shouldn't be at the root level then?

kks32 commented:
I'm happy for it to be moved outside the `classes` folder to the root level.

lharries commented:
I've updated the PR

Also moved `glyphtounicode.tex` from Classes to root (I assumed this was correct?), deleted the empty Classes folder, updated global references for it, and deleted the now unneeded troubleshoot for it

I've been building the project through overleaf so am not currently not able to compile it locally but it seems to have passed Travis. Perhaps worth checking manually too if possible?

P.S. thanks @kks32 for making/maintaining this, it's incredibly useful!
